content/docs/contributing/core.md

```diff
@@ -48,19 +48,20 @@ using `brew` to install Python. For Windows, we recommend an official
 
 <admon type="info">
 
-Note that `pip` version 20.3+ is required.
+Note that `pip` version 21.3+ is required.
```

Comment:
We now need `>=21.3` pip.

Reply:
But this note may be unnecessary tbh, I added `pip install -U pip` below which should suffice.